common/test/src/Model/TestGame.cpp

```diff
@@ -201,7 +206,7 @@ namespace TrenchBroom {
         }
 
         const Model::BrushFaceAttributes& TestGame::doDefaultFaceAttribs() const {
-            static const Model::BrushFaceAttributes defaults(Model::BrushFaceAttributes::NoTextureName);
+            static const Model::BrushFaceAttributes defaults(m_defaultFaceAttribs);
```

Comment:
BTW this change to the test code here means that the "defaults" returned by this function get locked into place the first time they are read. If you want to set the defaults to something then you have to call setDefaultFaceAttributes before they are used; any call to setDefaultFaceAttributes afterward will be ignored.

The more I think about this the more it bugs me a little. :-) I'm ok with leaving it this way, especially in test code, but if you want to suggest an alternative implementation I am all ears.